### ISSUE TITLE
Please update this plugin!

### ISSUE THREAD

**Issue by aegisvisuals** [thorwe/CrossTalk#58]
I've been wanting to use this plugin and see how it can help immerse my gaming, but I haven't been able to test it since it isn't up to date with the current version of TeamSpeak 3!

**Comment by aegisvisuals** [comment]
Yeah, I realized, still quite new to GitHub. However, thank you for
clarifying, have a nice day!

Den tors. 10. aug. 2023 kl. 15.02 skrev 24q ***@***.***>:

**Comment by aegisvisuals** [comment]
The radio or CrossTalk?

Den tors. 10. aug. 2023 kl. 15.12 skrev 24q ***@***.***>:

**Comment by xamionex** [comment]
Bump
